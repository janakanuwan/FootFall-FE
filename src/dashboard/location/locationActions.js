import { createAction } from 'redux-actions';
import { CHANGE_LOCATION } from "../../const/action-types";

const changeLocation = createAction(
  CHANGE_LOCATION,
  (selectedLocation) => ({ selectedLocation })
);

export { changeLocation };
