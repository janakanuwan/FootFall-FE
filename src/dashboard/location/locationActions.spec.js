import { changeLocation } from './locationActions';

describe('locationActions', () => {
  it('should create an action for location change', () => {
    expect(changeLocation({ id: 1, name: 'Test Location' })).toMatchSnapshot();
  });
});
