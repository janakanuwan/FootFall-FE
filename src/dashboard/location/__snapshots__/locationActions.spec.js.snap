// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`locationActions should create an action for location change 1`] = `
Object {
  "payload": Object {
    "id": 1,
    "name": "Test Location",
  },
  "type": "CHANGE_LOCATION",
}
`;
